rails/db/migrate/20240726071341_create_active_storage_tables.active_storage.rb

```
@@ -0,0 +1,57 @@
+      t.text     :metadata
+      t.string   :service_name, null: false
+      t.bigint   :byte_size,    null: false
+      t.string   :checksum
+
+      if connection.supports_datetime_with_precision?
+        t.datetime :created_at, precision: 6, null: false
+      else
+        t.datetime :created_at, null: false
+      end
+
+      t.index [ :key ], unique: true
```

rubocop

[convention] Do not use space inside array brackets.

(Layout/SpaceInsideArrayLiteralBrackets)